ketchup/main.py

```diff
@@ -1,121 +1,231 @@
+                except ValueError:
+                    print("Sorry, please only enter a value greater than 0.")
+                    continue
+                if episodes <= 0:
+                    print("Sorry, please only enter a value greater than 0.")
+                    continue
+                else:
+                    break
+
+            while True:
+                try:
+                    per_week_limit = int(input(
+                        "How many days per week can you "
+                        "dedicate to watching?: "))
+
+                except ValueError:
+                    print("Sorry, please only enter a value greater than 0.")
+                    continue
+
+                if per_week_limit <= 0:
+                    print("Sorry, please only enter a value greater than 0.")
+                    continue
+
+                else:
+                    break
+
+            weeks_required = math.ceil(episodes / per_week_limit)
+
+            episodes_per_day_alternative = 3
+            episodes_per_day = episodes_per_day_alternative * per_week_limit
+
+            weeks_required_alternative = math.ceil(episodes / episodes_per_day)
+
+            start_date_prompt = "When would like to begin watching? " \
+                                "(Example: 15 Aug 2050): "
+
+            start_date = get_date(start_date_prompt, require_future=True)
+
+            realistic_end_date = start_date + dt.timedelta(
+                weeks=weeks_required)
+
+            realistic_end_date_alternative = start_date + dt.timedelta(
+                weeks=weeks_required_alternative)
+
+            print(f"To watch {episodes} episodes at a rate of {per_week_limit} "
+                  f"days per week, you should finish by "
+                  f"{realistic_end_date.strftime('%d %b %Y')}.\n")
+            #  The realistic_end_date is considering only two episodes per week
+            #  being watched by the user.
+
+            print(f"Alternatively, you can watch {episodes_per_day_alternative}"
+                  f" episodes a day, which is "
+                  f"{episodes_per_day_alternative * per_week_limit} episodes "
+                  f"per week. So you could finish within approximately "
+                  f"{episodes_per_day_alternative} weeks.\n")
+
+            print(f"Watching {episodes_per_day_alternative * per_week_limit} "
+                  f"episodes per week instead, you can finish by "
+                  f"{realistic_end_date_alternative.strftime('%d %b %Y')}.\n")
+
+        elif user_option == 5:
+            print("Thank you for using AnimeKetchup. Goodbye!")
+            break
+
+        else:
+            print("Invalid option. Please choose a valid option.")
+
+        another_option = input(
+            "Would you like to choose another option? (Y/N): ").lower()
+        if another_option != "y":
+            print("Thank you for using AnimeKetchup. Goodbye!")
+            break
```

Comment:
Perhaps lets pull all "logic" out of main.py into separate but relevant file. Maybe we create an "options.py" that allows us to set what the different logic paths are that the user could take? and then that way the main function only asks a user to pick a number and then calls an option from there.

Reply:
Done, added to a new pull request.